tania-prof.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE hmtl>
+<hmtl>
+
+    <head>
+        <link rel="stylesheet" type="text/css" href="h1 {.css">
+        <!-- Add icon library -->
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+        <title>Tania Yeroman's Personal Profile</title>
+    </head>
+    <main>
+
+        <body>
+            <nav>
+                <div class="navbar">
+                    <a href=#>About</a>
+
+                    <a href=#>Experience</a>
+
+                    <a href=#>Contact</a>
+
+                    <a href=#>Home<a/>
+                </div>
+            </nav>
+            <h1>Tania Yeromiyan</h1>
+            <section class="flexcontainer">
+                <div class="column1">
+            
+
+            <img src= "imagen_de_ios1.jpg" width="300px" height="250px" alt="Tania">
+
+                    <h2 class="nameage"> Tania Yeromiyan,28 </h2>
+                    <div class="flex-container-socials">
+                    
+                        
+                            <a href="https://github.com/TaniaY21" class="fa fa-github"></a>
```

Comment:
Look into a linter such as Prettier which will automatically format code on save - which is lovely as one less thing for us to think about! Code formatting is great as it ensure tidy and easy to read files. There is a few sections with empty lines between elements and inconsistency in position. Research into code formatting and linters to help with this.